code/game/atoms.dm

```diff
@@ -802,7 +802,7 @@
   * Called when lighteater is called on this.
   */
 /atom/proc/lighteater_act(obj/item/light_eater/light_eater)
-	return
+	SEND_SIGNAL(src,COMSIG_ATOM_LIGHTEATER_ACT)
```

Comment:
```suggestion
	SHOULD_CALL_PARENT(TRUE)
	SEND_SIGNAL(src,COMSIG_ATOM_LIGHTEATER_ACT)
```

Reply:
just wondering what this would do, and why this should be done. since there is no parent to call with unless im mis understanding